hazelcast-tpc-engine/src/test/java/com/hazelcast/internal/tpcengine/AsyncSocketOptionsTest.java

```diff
@@ -160,7 +162,7 @@ public void test_SO_TIMEOUT() {
 
     @Test
     public void test_TCP_KEEPCOUNT() {
-        assumeIfNioThenJava11Plus();
+        assumeIfNioThenJava11PlusAndLinux();
```

Comment:
Can you update `NioAsyncSocketOptions` `*_PRINTED` variables then? For example this:
```
if (TCP_KEEPCOUNT_PRINTED.compareAndSet(false, true)) {
    logger.warning("Ignoring TCP_KEEPCOUNT. "
            + "Please upgrade to Java 11+ or configure tcp_keepalive_probes in the kernel: "
            + "For more info see https://tldp.org/HOWTO/html_single/TCP-Keepalive-HOWTO/. "
            + "If this isn't dealt with, idle connections could be closed prematurely.");
}
```

Reply:
Done